acceptance-tests/pom.xml

```diff
@@ -53,29 +53,6 @@
             <artifactId>selenium-firefox-driver</artifactId>
             <version>2.53.0</version>
         </dependency>
-        <!-- begin HttpRequest helper -->
-        <dependency>
-            <groupId>io.jenkins.blueocean</groupId>
-            <artifactId>blueocean-rest-impl</artifactId>
-            <version>1.4.0-beta-1-SNAPSHOT</version>
```

Comment:
LGTM. 

Yeah dependency from ath to snapshot version is always going to be a problem. Specially it the thing that it intends to test. Either we make it standalone released http test library or use the one already there.